src/kube-runtime/src/runtime

```diff
@@ -46,19 +46,21 @@ PAI_RUNTIME_LOG_FILE=${PAI_LOG_DIR}/user.pai.all
 # priority=0
 source ${PAI_RUNTIME_DIR}/runtime_env.sh
 
-# setup ssh
-# priority=1
-${PAI_RUNTIME_DIR}/sshd.sh >> ${PAI_RUNTIME_LOG_FILE} 2>&1
-
 # check port conflict
 python ${PAI_RUNTIME_DIR}/port.py >> ${PAI_RUNTIME_LOG_FILE} 2>&1
```

Comment:
This should redirect to runtime log, not user log

Reply:
Ok